composer.json

```diff
@@ -1,32 +1,61 @@
 {
   "name": "getsidekicker/flagr-feature-laravel",
+  "description": "Makes working with Flagr in Laravel a snap",
   "type": "library",
+  "license": "MIT",
+  "keywords": [
+    "flagr",
+    "feature",
+    "feature flag",
+    "laravel"
+  ],
+  "authors": [
+    {
+      "name": "Sidekicker",
+      "email": "product@sidekicker.com.au",
+      "role": "Developer"
+    }
+  ],
+  "support": {
+    "issues": "https://github.com/getsidekicker/flagr-feature-laravel/issues"
+  },
   "require": {
+    "php": "^8.0",
     "joshkulesza/php-flagr": "1.1.13.1",
-    "guzzlehttp/psr7": "^1.0"
+    "guzzlehttp/psr7": "^1.0",
+    "spatie/laravel-package-tools": "^1.9"
   },
   "require-dev": {
     "orchestra/testbench": "^6.19",
-    "phpunit/phpunit": "^9.5"
+    "phpunit/phpunit": "^9.5",
+    "friendsofphp/php-cs-fixer": "^3.0",
+    "squizlabs/php_codesniffer": "^3.6",
+    "phpstan/phpstan": "^0.12.94"
   },
   "autoload": {
     "psr-4": {
-      "Sidekicker\\Feature\\": "src"
+      "Sidekicker\\FlagrFeatureLaravel\\": "src/"
```

Comment:
**suggestion:** As discussed, it's odd to have "Laravel" in the namespace, FlagrFeature is better.

Reply:
Fixed